### ISSUE TITLE
snpMLST.py genotype error: KeyError: 'snp_pos'

### ISSUE THREAD

**Issue by snayfach** [snayfach/snpMLST-issues#9]
I ran this command: `snpMLST.py genotype --db example/array/test/test-kmer-db.json -1 unit_test/hts_reads/SRR413665_2.fastq.gz --out test.genotype`

And got this error:

```
[load] loading kmer databases from disk: example/array/test/test-kmer-db.json
    loading is done, current RAM: 6177.21 MB
	elapsed time: 10.86 seconds

[search] Searching kmer database
Traceback (most recent call last):
  File "/Users/snayfachbattilana/Desktop/snpMLST/scripts/snpMLST.py", line 82, in <module>
    map_reads.main()
  File "/Users/snayfachbattilana/Desktop/snpMLST/snp_mlst/map_reads.py", line 621, in main
    search_kmers(args, kmer_db)
  File "/Users/snayfachbattilana/Desktop/snpMLST/snp_mlst/map_reads.py", line 398, in search_kmers
    fw.write("{} {}\n".format(k, v['snp_pos'][0]))
KeyError: 'snp_pos'
```

**Comment by zjshi** [comment]
Couldn't reproduce the error in both environments, would you please share some content from the "example/array/test/test-kmer-db.json"?

**Comment by snayfach** [comment]
You should have received a dropbox link with some files. Let me know what
else I can do to help

**Comment by snayfach** [comment]
The issue was that I was specifying the wrong database input file. Closing this for now. 

In the future, we should add a function to each module that checks for valid inputs. I did this quite a bit with my other software packages and I think that users appreciate it. For example, if the input is not valid (file does not exist or incorrect file type or extension) then the program makes a controlled exit and reports the error.
